src/libnnstreamer-edge/nnstreamer-edge-internal.c

```diff
@@ -1327,7 +1424,6 @@ nns_edge_create_handle (const char *id, nns_edge_connect_type_e connect_type,
     }
   }
 
```

Comment:
This is not the code that was added this time,
I think it would be good to extract this part as an additional function. What do you think about it? :smile: 

```
if (NNS_EDGE_CONNECT_TYPE_AITT == connect_type) {
    ret = initialize_aitt_handle (eh);
    if (NNS_EDGE_ERROR_NONE != ret) {
      return ret;
    }
  }
``` 
```
static int
initialize_aitt_handle (nns_edge_handle_s * eh)
{
  int ret = NNS_EDGE_ERROR_NONE;

  ret = nns_edge_aitt_create (&eh->broker_h);
  if (NNS_EDGE_ERROR_NONE != ret) {
    nns_edge_loge ("Failed to create AITT handle.");
    nns_edge_release_handle (eh);
    return ret;
  }
```
  return NNS_EDGE_ERROR_NONE;
}
```

Reply:
Oh, I see. I'm looking forward to the new service too. Thank you for the detailed explanation. :pray: